config/dev.webpack.config.js

```diff
@@ -2,6 +2,7 @@ const { resolve } = require('path');
 const config = require('@redhat-cloud-services/frontend-components-config');
 const { config: webpackConfig, plugins } = config({
     rootFolder: resolve(__dirname, '../'),
+    sassPrefix: '.ros, .inventory',
```

Comment:
Unless you are importing inventory component here you don't really need to add the `.inventory` part.

Reply:
@karelhala, I tried by removing `.inventory` class from sassPrefix.

1. After change applied, I am getting color of empty icon black instead of green -
![Screenshot from 2021-05-24 18-29-15](https://user-images.githubusercontent.com/6470528/119353642-c3397f00-bcc0-11eb-831c-d4fa886a3984.png)


2. After adding `.inventory` class then it shows green icon -

![Screenshot from 2021-05-24 18-40-45](https://user-images.githubusercontent.com/6470528/119353065-0e9f5d80-bcc0-11eb-8d31-5570508c5453.png)

In case (1), it is not applying ros specific CSS changes. 
In case(2) it adds extra CSS path using `.inventory`. 
Any pointers? Not sure which one is the expected behaviour.

Reply:
Mhh, I thought that the `sassPrefix` was affecting the styling from app where it's used. But this proves me wrong, keep as is then!

Reply:
Done!